lib/adapters/leveldb/leveldb.js

```diff
@@ -1192,9 +1192,6 @@ function LevelPouch(opts, callback) {
 
       revs.forEach(function (rev) {
         var seq = seqs[rev];
-        if (!seq) {
-          return;
-        }
```

Comment:
`seq` should never be 0 or undefined or null. And even if it is, this should be `return checkDone(someError)`. I prefer to just remove it.